ontologies/vocabs.ttl

```diff
@@ -1148,8 +1130,10 @@ Tag for the government local industry sector"""^^xsd:string ;
 
 gist:_IndustrySector_government-National
 	a gist:IndustrySector ;
-	rdfs:label "Government National"^^xsd:string ;
-	skos:altLabel "Government-National"^^xsd:string ;
+	skos:altLabel
+		"Government National"^^xsd:string ,
+		"Government-National"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?